config/crd/bases/mongodbcommunity.mongodb.com_mongodbcommunity.yaml

```diff
@@ -5,12 +5,6 @@ kind: CustomResourceDefinition
 metadata:
   annotations:
     controller-gen.kubebuilder.io/version: v0.4.1
-    service.binding/type: 'mongodb'
```

Comment:
I remember a discussion that we want to keep that in the crd base, albeit make removes it

Reply:
I have added those to our types and upgraded the Controller Gen to pick them up.